src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -920,9 +920,9 @@ public DynamicJsonArray GetClusterErrors()
             return _engine.GetClusterErrorsFromLeader();
         }
 
-        public async Task<string> GenerateClusterIdentityAsync(string id, string databaseName)
+        public async Task<(long clusterEtag, string clusterId)> GenerateClusterIdentityAsync(string id, string databaseName)
```

Comment:
I think that it should be uppercase. Like with public fields/properties in classes.